assets/src/components/detours/detourModal.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import { DiversionPage } from "./diversionPage"
+import { StartDetourProps } from "./detourDropdown"
+
+export const DetourModal = ({
+  detourInfo,
+  onClose,
+  show,
+}: {
+  detourInfo: StartDetourProps
+  onClose: () => void
+  show: boolean
+}) => {
```

Comment:
**nitpick (non-blocking):** since this function body is of the form

```
{
  return (
    < ...JSX Code... />
  )
}
```

you can rewrite it as a simple value without a `return` statement or curly braces.